server/test/unit/test_repo_distributor_manager.py

```diff
@@ -420,13 +420,16 @@ def test_update_validate_exception(self):
         distributor = self.distributor_manager.add_distributor('elf', 'mock-distributor', {}, True)
         dist_id = distributor['id']
 
-        mock_plugins.MOCK_DISTRIBUTOR.validate_config.side_effect = Exception()
+        class TestException(Exception):
+            pass
+
+        mock_plugins.MOCK_DISTRIBUTOR.validate_config.side_effect = TestException()
 
         # Test
         try:
             self.distributor_manager.update_distributor_config('elf', dist_id, {})
             self.fail('Exception expected')
-        except exceptions.PulpDataException:
+        except TestException:
```

Comment:
If you feel like cleaning this up a touch, even though it's not directly related to your bug fix, it's a good use case for ``self.assertRaises``